README.md

```diff
@@ -126,6 +126,10 @@ or
 ```shell
 $ make CROSS_COMPILE=arm-linux-gnueabihf- check # ARMv7-A
 ```
+or
+``` shell
+$ make CROSS_COMPILE=arm-linux-gnueabihf- A32=1 check # ARMv8-A running in 32-bit mode, i.e., A32 instruction set
```

Comment:
In this case, A32 is always 1, right?

Reply:
Correct. A32 always sets to 1 if we want to run test program compiled under A32 instruction set.

Reply:
If toolchain is set to `arm-linux-gnueabihf-`, `A32` should not appear since the former implies the latter.

Reply:
The goal of using `A32` for indicating A32 instruction set build is that we can not only test our code on AArch32 but also check whether the NEON optimization on A32 will create any impacts or not.

Nevertheless, we may set the `-mcpu` flag in other Makefile input parameters such as `CFLAGS`, and I consider I can start to implement from this.